main/User_config.h

```diff
@@ -101,6 +101,7 @@ char mqtt_port[6] = "1883";
 //#define ZsensorGPIOKeyCode "GPIOKeyCode" //ESP8266, Arduino, ESP32
 //#define ZsensorGPIOInput "GPIOInput" //ESP8266, Arduino, ESP32
 //#define ZmqttDiscovery "HADiscovery"//ESP8266, Arduino, ESP32, Sonoff RF Bridge
+//#define ZactuatorFASTLED "FASTLED"  //ESP8266, Arduino, ESP32, Sonoff RF Bridge
```

Comment:
Could you add this line also to test_config.h, this way the Continuous integration will test your changes.
You will need also to add the library call in the platformio.ini file.

Reply:
add `ZactuatorFASTLED ` only for esp8266 or to all?

Reply:
I insert it to all...